package-lock.json

```diff
@@ -4,6 +4,16 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
+    "2-thenable": {
```

Comment:
Should this be a `yarn.lock` file instead of a `package-lock.json` file?

Reply:
the open source path has been with npm. i'll make that adjustment as we build a new alternative here to open source.

Reply:
fyi - decided to transition to yarn in the end per suggestion above